ios/Classes/SwiftSumupPlugin.swift

```diff
@@ -204,6 +204,10 @@ public class SwiftSumupPlugin: NSObject, FlutterPlugin {
     private func wakeUpTerminal() {
         SumUpSDK.prepareForCheckout()
     }
+
+    private func prepareForCheckout() {
```

Comment:
This method should be called from the `handle` method, otherwise it will never be used

Reply:
Sorry for the slip